tests/core/test_oauth_clients.py

```diff
@@ -0,0 +1,58 @@
+from flask import json
+from newsroom.tests.users import (test_login_succeeds_for_admin, init as user_init)  # noqa
+from superdesk import get_resource_service
+import base64
+
+
+def test_oauth_clients(client):
+    test_login_succeeds_for_admin(client)
+    # Register a new client
+    response = client.post(
+        "/oauth_clients/new",
+        data=json.dumps({"name": "client1"}),
+        content_type="application/json",
+    )
+    assert response.status_code == 201
+
+    oauth_client = get_resource_service("oauth_clients").find_one(
+        req=None, name="client1"
+    )
+
+    # Check for the client secret
+    password = response.json.get("password", None)
+    if not password:
```

Comment:
it's better to do `assert password is not None` so you get some hint in case there is an error